prow/scripts/cluster-integration/kyma-gke-long-lasting.sh

```diff
@@ -157,17 +157,10 @@ function waitUntilInstallerApiAvailable() {
 
 function installKyma() {
 
-	kymaUnsetVar=false
-
-	for var in REMOTEENVS_IP_ADDRESS GATEWAY_IP_ADDRESS ; do
-    	if [ -z "${!var}" ] ; then
-        	echo "ERROR: $var is not set"
-        	kymaUnsetVar=true
-    	fi
-	done
-	if [ "${kymaUnsetVar}" = true ] ; then
-    	exit 1
-	fi
+  if [ -z "$GATEWAY_IP_ADDRESS" ] ; then
```

Comment:
Please leave the current logic, as it is common in all of the scripts, just trim the unused envs